docs/Admin.md

```diff
@@ -27,31 +33,156 @@ export default App;
+        dataProvider={dataProvider}
+        authProvider={authProvider}
+        i18nProvider={i18nProvider}
+        dashboard={Dashboard}
+        loginPage={Login}
+        layout={Layout}
+        theme={lightTheme}
+        darkTheme={darkTheme}
+        defaultToLightTheme
+    >
+        <Resource {...customers} />
+        <Resource {...orders} />
+        <Resource {...invoices} />
+        <Resource {...products} />
+        <Resource {...categories} />
+        <Resource {...reviews} />
+        <CustomRoutes>
+            <Route path="/segments" element={<Segments />} />
+        </CustomRoutes>
+    </Admin>
+);
+```
+
+## Props
+
+Three main props lets you configure the core features of the `<Admin>` component:
+
+- [`dataProvider`](#dataprovider) for data fetching
+- [`authProvider`](#authprovider) for security and permissions
+- [`i18nProvider`](#i18nprovider) for translations and internationalization
+
 Here are all the props accepted by the component:
 
-- [`dataProvider`](#dataprovider)
-- [`authProvider`](#authprovider)
-- [`i18nProvider`](#i18nprovider)
-- [`queryClient`](#queryclient)
-- [`title`](#title)
-- [`dashboard`](#dashboard)
-- [`disableTelemetry`](#disabletelemetry)
-- [`catchAll`](#catchall)
-- [`menu`](#menu)
-- [`theme`](#theme)
-- [`layout`](#layout)
-- [`loginPage`](#loginpage)
-- [`authCallbackPage`](#authcallbackpage)
-- [`history`](#history)
-- [`basename`](#basename)
-- [`ready`](#ready)
-- [`requireAuth`](#requireauth)
-- [`store`](#store)
-- [`notification`](#notification)
+| Prop                   | Required | Type           | Default        | Description                                              |
+|------------------------|----------|----------------|----------------|----------------------------------------------------------|
+| `dataProvider`         | Required | `DataProvider` | -              | The data provider for fetching resources                 |
+| `children`             | Required | `ReactNode`    | -              | The routes to render                                     |
+| `authCallbackPage`     | Optional | `Component`    | `AuthCallback` | The content of the authentication callback page          |
+| `authProvider`         | Optional | `AuthProvider` | -              | The authentication provider for security and permissions |
+| `basename`             | Optional | `string`       | -              | The base path for all URLs                               |
+| `catchAll`             | Optional | `Component`    | `NotFound`     | The fallback component for unknown routes                |
+| `dashboard`            | Optional | `Component`    | -              | The content of the dashboard page                        |
+| `darkTheme`            | Optional | `object`       | -              | The dark theme configuration                             |
+| `defaultTo LightTheme` | Optional | `boolean`      | `false`        | Flag to default to the light theme                       |
```

Comment:
```suggestion
| `defaultToLightTheme` | Optional | `boolean`      | `false`        | Flag to default to the light theme                       |
```

Reply:
This was made on purpose to add a line break and make the table look okayish despite the large content